packages/docs/tutorial/tutorial-start.yaml

```diff
@@ -45,243 +40,246 @@ _ref:
       #         src="https://www.youtube.com/embed/Cd4Xxxisykg" frameborder="0"
       #         allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
       #       </iframe>
-      # - id: body_start
-      #   type: MarkdownWithCode
-      #   properties:
-      #     content: |
-      #       In this tutorial, we will be creating a simple ticketing app, that allows users to file new tickets and see a list of outstanding tickets. The app will write the ticket data to Google Sheets, and we will deploy the app using [Netlify](https://www.netlify.com).
 
-      #       ### Requirements
+      - id: body_start
+        type: MarkdownWithCode
+        properties:
+          content: |
+            In this tutorial, we will be creating a simple ticketing app that allows users to file new tickets and see a list of outstanding tickets. The app reads product data from [DummyJSON](https://dummyjson.com/) and will write the ticket data to an [SQLite](https://www.sqlite.org/) database.
 
-      #       The Lowdefy CLI (Command Line Interface) is needed to run the development server. To run the Lowdefy CLI you need to install Node.js at version 12 or greater. If you don't have it installed, download Node.js from https://nodejs.org/en/download/, and follow the installation steps for your computer. All of the default settings given by the installer are fine for what we need. You will also need a text editor to modify the Lowdefy configuration files.
+            ### Requirements
 
-      #       #### Step 1.1 - Create a project directory
+            The Lowdefy CLI (Command Line Interface) is needed to run the development server. To run the Lowdefy CLI you need to install Node.js at version 12 or greater. If you don't have it installed, download Node.js from https://nodejs.org/en/download/, and follow the installation steps for your computer. All of the default settings given by the installer are fine for what we need. You will also need a text editor to modify the Lowdefy configuration files.
 
-      #       Create a directory (folder) on your computer where you would like to create the configuration files for your project. We will be referring to this directory as the project directory
+            #### Step 1.1 - Create a project directory
```

Comment:
Lowdefy needs to have pnpm installed for V4. So we need to add a section on that. Think we should direct users to run `corepack enable`, but also link to other ways to install pnpm (https://pnpm.io/installation)

Reply:
Change made in 03779d5cfac0a1aa60c314008ea9419ede966cff